api/challenge_api/Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
+# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
```

Comment:
We can remove these comments. We're safe to assume that Rails Engineers understand how to use this file.

Reply:
Oddly enough, I can't find the Rakefile in my file tree. Is there any reason you can think of that a file would be missing/hiding?

Reply:
The only thing I can think of is maybe your local copy of the repo is out of sync with the remote copy? As long as the Rakefile file isn't in the .gitignore, I can't think of any reason why it would be missing.